save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/testsuiteselector/TestSuiteSelectorSearchMode.kt

```diff
@@ -61,6 +63,30 @@ private fun ChildrenBuilder.buildInput(
     }
 }
 
+private fun ChildrenBuilder.showAvailableTestSuitesForSearchMode(
+    testSuites: List<TestSuiteDto>,
+    selectedTestSuites: List<TestSuiteDto>,
+    isOnlyLatestVersion: Boolean,
+    onTestSuiteClick: (TestSuiteDto) -> Unit,
+) {
+    val testSuitesToBeShown = testSuites.filter {
+        !isOnlyLatestVersion || it.version == it.source.latestVersion
+    }
+
+    // small hack to hide version if only latest is shown
```

Comment:
Do we need to hide it though?

Reply:
Well, I think so. I try not to show too many labels if possible

Reply:
I think it's confusing that at first you see versions and then after you tick 'Show only latest', the version disappears. I think it's better to keep seeing the exact version you are about to select

Reply:
Ok, we might want to display version regardless the checkbox, will fix it

Reply:
Done